packages/compass/src/app/components/home.tsx

```diff
@@ -189,6 +189,19 @@ export default function ThemedHome(
           });
         }
       }}
+      onContextMenuOpen={(itemGroups) => {
+        if (itemGroups.length > 0) {
+          track('Context Menu Opened', {
+            item_groups: itemGroups.map((group) => group.telemetryLabel),
+          });
+        }
+      }}
```

Comment:
This is slightly annoying, but this only adds tracking on the compass desktop side, you'd want the same code on the compass-web side too (until we have time to deal with https://jira.mongodb.org/browse/COMPASS-7830)

Reply:
created https://jira.mongodb.org/browse/CLOUDP-336743